packages/legend-graph/src/graphManager/action/EngineError.ts

```diff
@@ -34,6 +34,10 @@ export class EngineError extends ApplicationError {
   setSourceInformation(sourceInformation: SourceInformation | undefined): void {
     this.sourceInformation = sourceInformation;
   }
+
+  get nonEmptyMessage(): string {
```

Comment:
don't think we need this, let's just hard code the error in various places with the message `(no error message)`

Reply:
will remove this